references/tags.tsv

```diff
@@ -198,6 +198,7 @@ Seide2014_parallel	doi:10.1109/ICASSP.2014.6853593
 Setty2015_seqgl	doi:10.1371/journal.pcbi.1004271
 Selvaraju2016_grad	arxiv:1610.02391
 Shaham2016_batch_effects	arxiv:1610.04181
+Shapely_wikipedia	url:https://en.wikipedia.org/wiki/Shapley_value
```

Comment:
@dhimmel I can't find a suitable URL for this.  A DOI seems unlikely for a reference from the 50s.  I'm using `url:http://oai.dtic.mil/oai/oai?verb=getRecord&metadataPrefix=html&identifier=AD0604084` for now and will keep hunting.